packages/libpostal/test/expand.spec.js

```diff
@@ -0,0 +1,328 @@
+import ezs from '../../core/src';
+import expandAddress from '../src/expand-address';
+import expandAddressWith from '../src/expand-address-with';
+import runEzs from '../test-utils/runEzs';
+
+ezs.addPath(__dirname);
+
+describe('expandAddress, expandAddressWith', () => {
+
+    beforeAll(() => {
+        ezs.use({ expandAddress, expandAddressWith });
+    });
+
+    describe('expandAddress', () => {
+        const wrongData = [
+            {'hello': 'world'},
+            {'mario': 'luigi'}
+        ];
+
+        const wrongData2 = [
+            [
+                {'hello': 'world'},
+                {'mario': 'luigi'}
+            ],
+            [
+                {'wario': 'waluigi'}
+            ]
+        ];
+
+        const simpleData =  [
+            'Barboncino 781 Franklin Ave, Crown Heights, Brooklyn, NY 11238',
+            'Inist-CNRS 2, rue Jean Zay CS 10310 F‑54519 Vandœuvre-lès-Nancy France'
+        ];
+
+        const simpleData2 =  [
+            [
+                'Barboncino 781 Franklin Ave, Crown Heights, Brooklyn, NY 11238',
+                'Inist-CNRS 2, rue Jean Zay CS 10310 F‑54519 Vandœuvre-lès-Nancy France'
+            ],
+            [
+                'University of Bordeaux, IMS, CNRS UMR5218, Talence, F-33405, France'
+            ]
+        ];
+
+        it('should expandAddress return the same object', async () => {
```

Comment:
<!-- Polish comments are like a suggestion, where there is nothing necessarily wrong with the relevant content, there's just some ways to immediately improve the quality.  -->
**polish:** ‏Don't repeat the function name, it is already present in the `describe`
```suggestion
        it('should return the same object', async () => {
```